### DISCUSSION TITLE
请问在JNI_ONLOAD里注册的JNI函数能hook么

### DISCUSSION THREAD

**Discussion by threeloe** [bytedance/bhook#89] [Q&A]
如果不能的话能帮忙提供下思路么，我想hook下libhwui中的android_view_ThreadedRenderer_setStopped，各种测试都hook不成功，找不到符号

**Comment by Colibrow** [comment]
可以尝试找一下对应的ArtMethod指针然后替换data_的值做一个JNI Hook
